tests/browser/story-screenshots.test.ts

```diff
@@ -27,8 +27,15 @@ Object.values(stories).forEach((story) => {
     Object.keys(themes).forEach((theme) => {
       if (themes[theme].incomplete) return;
 
+      if (theme !== 'core' && story.importPath.includes(themes[theme].packageName)) return;
+
       // Don't take screenshots of theme-specific components outside of their themes
-      if (isSmokeTest && !story.importPath.includes(themes[theme].packageName)) return;
+      // during smoke tests.
+      if (
+        isSmokeTest &&
+        !(theme === 'core' || !story.importPath.includes(themes[theme].packageName))
+      )
+        return;
```

Comment:
[Prior to the introduction of interaction tests](https://github.com/CMSgov/design-system/blob/753fddf3ef69c55b66a605a3b58faecb316db528/tests/browser/story-screenshots.test.ts), this if-statement had the following comment, which more accurately describes what the logic is there for:

> // For smoke tests, only capture core components in the core theme and theme-specific
> // components in their native theme

The comment that is currently above it goes above the line you're adding now, which is

> Don't take screenshots of theme-specific components outside of their themes

Reply:
And thanks for simplifying the different checks down to `story.importPath.includes(themes[theme].packageName))`!

Reply:
Good call 👍🏻 Updated